bin/diagnostic.js

```diff
@@ -1 +1,13 @@
-'use strict';
+const ask = require('../lib/ask.js');
+'use strict'
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret!");
+  if (answer > 0 && answer < 11) {
+   console.log("You guessed it!");
```

Comment:
This causes ```You guessed it!``` to print for any input within the valid range.  There is also no breaking condition so this is an infinite loop.